docs/metrics-howto.md

```diff
@@ -60,6 +60,31 @@
 
 1.  Restart Prometheus.
 
+## Monitoring workers
+
+To monitor a Synapse installation using
+[workers](https://github.com/matrix-org/synapse/blob/master/docs/workers.md),
+every worker needs to be monitored independently, in addition to
+the main homeserver process. This is because workers don't send
+their metrics to the main homeserver process, and therefore
+expose their own metrics themselves (if they are configured to
+do so).
```

Comment:
I find the last sentence confusing, I think it can be simplified:

> This because work don't send their metrics to the main homeserver process, but expose them directly (if they are configured to do so).

Reply:
Much simpler indeed. Fixed in https://github.com/matrix-org/synapse/pull/7357/commits/236dbba7e323e38f6bb4264e75970823cc8d7bec